script/dryrun

```diff
@@ -32,6 +33,7 @@ SKIP = {
     "sql/telemetry/core_clients_last_seen_raw_v1/init.sql",
     "sql/telemetry/fxa_users_last_seen_raw_v1/init.sql",
     # Reference table not found
+    "sql/monitoring/structured_detailed_error_counts_v1/view.sql",
```

Comment:
This sounds like an error we care about. Do we understand why this needs to be exempted?

Reply:
Right now the view doesn't have access to the `payload_bytes_decoded` dataset. We do certainly care about that, cc @whd for making this an authorized view.

Reply:
Ah, this error in particular is actually because it references the other view. Offhand I don't see an easy way to dry-run a DAG of queries.